oracle/datadog_checks/oracle/oracle.py

```diff
@@ -61,9 +62,11 @@ class Oracle(AgentCheck):
             ('PGA_MAX_MEM', 'oracle.process.pga_maximum_memory'),
         ]
     )
+    use_oracle_client = True
+    server = None
+    service_check_tags = None
```

Comment:
These would affect every instance. Let's move elsewhere

Reply:
I don't understand what you mean. It just provides the default of the variables? There are overridden at the instance level.

Reply:
Removed!